package.json

```diff
@@ -13,13 +13,14 @@
     "migrate": "sequelize db:migrate",
     "migrate:test": "sequelize db:migrate:undo:all && npm run migrate",
     "postinstall": "npm run dev",
-    "test": "export NODE_ENV=test && npm run migrate:test && nyc --reporter=html --reporter=text mocha --recursive --require babel-register --exit",
+    "test": "export NODE_ENV=test && npm run migrate:test && nyc --reporter=html --reporter=text mocha --recursive --require babel-register --exit --timeout 20000",
```

Comment:
Why are you increasing the test timeout to 20s per block?

Reply:
There was no timeout specified for tests and it is causing some tests to fail because the promises used.

Reply:
Alright. 👍